homeassistant/components/darksky/sensor.py

```diff
@@ -384,23 +387,55 @@
     "alerts": ["Alerts", None, None, None, None, None, "mdi:alert-circle-outline", []],
 }
 
-CONDITION_PICTURES = {
-    "clear-day": ["/static/images/darksky/weather-sunny.svg", "mdi:weather-sunny"],
-    "clear-night": ["/static/images/darksky/weather-night.svg", "mdi:weather-night"],
-    "rain": ["/static/images/darksky/weather-pouring.svg", "mdi:weather-pouring"],
-    "snow": ["/static/images/darksky/weather-snowy.svg", "mdi:weather-snowy"],
-    "sleet": ["/static/images/darksky/weather-hail.svg", "mdi:weather-snowy-rainy"],
-    "wind": ["/static/images/darksky/weather-windy.svg", "mdi:weather-windy"],
-    "fog": ["/static/images/darksky/weather-fog.svg", "mdi:weather-fog"],
-    "cloudy": ["/static/images/darksky/weather-cloudy.svg", "mdi:weather-cloudy"],
-    "partly-cloudy-day": [
-        "/static/images/darksky/weather-partlycloudy.svg",
-        "mdi:weather-partly-cloudy",
-    ],
-    "partly-cloudy-night": [
-        "/static/images/darksky/weather-cloudy.svg",
-        "mdi:weather-night-partly-cloudy",
-    ],
+
+class ConditionPicture(NamedTuple):
+    """Entity picture and icon for condition."""
+
+    entity_picture: str
+    icon: str
+
+
+CONDITION_PICTURES: dict[str, ConditionPicture] = {
```

Comment:
Can't the type checker infer this?

Reply:
https://github.com/home-assistant/core/pull/56941#discussion_r720801343